verifiers/internal/gha/slsaprovenance/v1.0/byob.go

```diff
@@ -0,0 +1,265 @@
+package v1
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	intoto "github.com/in-toto/in-toto-golang/in_toto"
+	slsa1 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v1"
+	serrors "github.com/slsa-framework/slsa-verifier/v2/errors"
+
+	"github.com/slsa-framework/slsa-verifier/v2/verifiers/internal/gha/slsaprovenance/common"
+)
+
+var (
+	// byobBuildType is the base build type for BYOB delegated builders.
+	byobBuildType = "https://github.com/slsa-framework/slsa-github-generator/delegator-generic@v0"
+
+	// containerBasedBuildType is the build type for the container-based builder and is based on BYOB.
+	containerBasedBuildType = "https://slsa.dev/container-based-build/v0.1?draft"
+
+	// genericGHABuildType is used by some tests.
+	// TODO: Update tests to use a real buildType
+	genericGHABuildType = "https://github.com/Attestations/GitHubActionsWorkflow@v1"
```

Comment:
I wonder if we need to support this buildType or if this was just used for tests.

Reply:
i think it's an old type we dont use, and probably can remove it